v2/src/components/VCard.vue

```diff
@@ -246,18 +246,14 @@ const hasDetails = computed(() => {
     align-self: flex-start;
     flex: 1;
     padding: spacing(3) spacing(6) spacing(6) spacing(3);
-    overflow: hidden;
-    .card-tags {
-      display: flex;
-      flex-direction: row;
-      flex-wrap: wrap;
-      gap: spacing(2);
-      margin-bottom: spacing(2);
-    }
     .card-title {
+      line-height: var(--font-size-8);
+      .v-tag {
+        float: left;
+        margin-right: spacing(2);
+        line-height: var(--font-size-8);
```

Comment:
you will have to update the margin-top and margin-right based on the font size the tag is next to ... so it stays baseline to the title and the desired distance on the right based on the design: 
![image](https://user-images.githubusercontent.com/13803343/176040934-89910bdd-0a90-42e1-98e2-ad5d3245467d.png)